python/adbc_driver_manager/adbc_driver_manager/dbapi.py

```diff
@@ -926,6 +926,22 @@ def fetch_df(self) -> "pandas.DataFrame":
             )
         return self._results.fetch_df()
 
+    def fetch_record_batch(self) -> pyarrow.RecordBatchReader:
+        """
+        Fetch a RecordBatchReader akin to
+        https://duckdb.org/docs/guides/python/export_arrow.html#export-as-a-recordbatchreader
```

Comment:
```suggestion
        Fetch the result as a PyArrow RecordBatchReader.
        
        This implements a similar API as DuckDB:
        https://duckdb.org/docs/guides/python/export_arrow.html#export-as-a-recordbatchreader
```

Consistency with other methods